python_modules/dagster/dagster_tests/core_tests/resource_tests/test_struct_config_resources.py

```diff
@@ -319,3 +322,47 @@ def another_asset(context: OpExecutionContext):
     defs.get_implicit_global_asset_job_def().execute_in_process()
 
     assert executed["yes"]
+
+
+def test_env_var():
```

Comment:
In a follow up you can use `environ` defined in test utils. To handle the context manager.

I also think we should rename `environ` do `set_env_vars_in_scope` or something to make it more greppable and obvious what is going on